cluster/saltbase/salt/kube-proxy/default

```diff
@@ -25,4 +25,13 @@
   {% endif -%}
 
 {% endif -%}
+
+{% set test_args = "" -%}
+{% if pillar['kubeproxy_test_args'] is defined -%}
+  {% set test_args=pillar['kubeproxy_test_args'] %}
+{% endif -%}
+
 DAEMON_ARGS="{{daemon_args}} {{api_servers_with_port}} {{kubeconfig}} {{pillar['log_level']}}"
+
+# test_args has to be kept at the end, so they'll overwrite any prior configuration
+DAEMON_ARGS="$DAEMON_ARGS + {{test_args}}"
```

Comment:
this is illegal on a systemd setup since environment files are not truly bash.

In this case, it prevented my Kubelet from getting any command line arguments.

I would prefer we change this to just append at the end of the above line.

Will submit a PR.